src/lib/sensors.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  BleSensors,
+  PowerMeter,
+  HeartRateMonitor,
+  CadenceMeter,
+  // @ts-ignore
+} from 'react-native-cycling-sensors';
+
+export const ble = new BleSensors();
```

Comment:
I've pulled these singletons out into their own module. We should discuss if we need these in a context provider or singletons as they currently are.